python/ray/util/sgd/tests/test_torch.py

```diff
@@ -9,6 +9,7 @@
 from torch.utils.data import DataLoader
 
 import ray
+import ray.util.sgd.data.examples.mlp_identity
```

Comment:
would it be possibe for you to just import the function you want instead of the entire module?

Reply:
So the functions I want include model_creator and optimizer_creator. How should I got about importing those shadowing the existing names?

`from ray.util.... import model_creator as model_creator_dataset` feels a little funky.